step.yml

```diff
@@ -27,57 +45,50 @@ toolkit:
 inputs:
   - aab_path:
     opts:
-      title: "The path of the AAB."
-      summary: Summary. No more than 2-3 sentences.
+      title: "Android App Bundle path"
+      summary: "Android App Bundle file (`.aab`) path"
       description: |
-        Description of this input.
-
-        Can be Markdown formatted text.
+        The **Android App Bundle path** input field is automatically filled out by the output of the previous build Step
       is_expand: true
       is_required: true
   - keystore_url: $BITRISEIO_ANDROID_KEYSTORE_URL
     opts:
-      title: "Keystore url"
-      summary: ""
-      description: |-
-        TO BE FILLED
+      title: "Keystore URL"
+      summary: "The keystore file's URL which is generated when you upload the file to the Code Signing tab."
+      description: ""
```

Comment:
The description should include the same summary if not more info. Did not enfoce this everywhere previously. The workflow editor UI displays the description when expanding a step input (not really consistently.) Or would reccomend modifying the workflow editor to fall back to the summary in an expanded view.